system/jlib/jlog.cpp

```diff
@@ -1737,9 +1828,19 @@ void CLogMsgManager::doReport(const LogMsg & msg) const
 {
     try
     {
-        ReadLockBlock block(monitorLock);
-        ForEachItemIn(i, monitors)
-            monitors.item(i).processMessage(msg);
+        switch (msg.queryCategory().queryClass())
```

Comment:
I wonder if you should use a new audience of internal as well, so you can quickly isolate these new messages